pkg/discovery/dtofactory/general_builder.go

```diff
@@ -33,6 +33,7 @@ var (
 		metrics.StorageAmount:      proto.CommodityDTO_STORAGE_AMOUNT,
 		metrics.VCPUThrottling:     proto.CommodityDTO_VCPU_THROTTLING,
 	}
+	vcpuUtilThreshold = 5.0
```

Comment:
I may miss previous discussions around this threshold. Have we considered making it configurable or just leave it as a fixed value?

Reply:
Good question.  For the time being as this is a global probe-wise config, it does make sense to make it configurable.  However, we're considering to adjust the threshold individually based on say the limit of the container.  With that in mind, my thought process is not to externalize the parameter at the moment to minimize handling of migration etc.  What do you think?

Reply:
I see. Yeah, it would better and smarter to have the threshold adjusted based on the limit of the container. Sounds good to start with such small utilThreshold value and collect more customers' feedback to see how it works.

Nit: would it be better to name this variable as `vcpuThrottlingUtilThreshold` to clearly indicate this is for VCPUThrottling not VCPU utilization?

Reply:
Yes totally!  I thought that's what I had; thanks for pointing it out.  Changed.

Reply:
Thanks Cheuk!